diagnostic.md

```diff
@@ -21,13 +21,18 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts
 ```
 
 Write an AJAX request to retrieve a list of **all donuts** on the server.
 
 ```js
-let getDonuts = /* your answer here */;
+let getDonuts = function () {
+  return $.ajax({
+    url: config.apiOrigin + '/donuts',
+    method: 'GET',
```

Comment:
We're looking for a hard coded url here, you can't assume you have config and apiOrigin variables established. 

```js
let getDonuts = function () {
  return $.ajax({
    url: 'http://www.example.com/donuts',
    method: 'GET',
  });
};
```